apps/picsa-apps/dashboard/src/app/modules/crop-information/pages/new_entry/new_entry.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="page-content">
+  <button mat-stroked-button color="primary" (click)="routeBack()"><mat-icon> Back </mat-icon></button>
```

Comment:
nit(non-blocking) - appreciate the consideration for managing back navigation, although I think it would probably be better if I implement some sort of breadcrumb/common navigation to use across all pages instead of relying on each page to implement - so can probably just remove here for now and I'll put on my todo list